lib/common/schemas.c

```diff
@@ -650,8 +652,53 @@ validate_xml(xmlNode *xml_blob, const char *validation, gboolean to_logs)
 }
 
 #if HAVE_LIBXSLT
+
+/* see crm_xml_error */
+static void
+crm_xslt_err(void *ctx, const char *msg, ...)
```

Comment:
I don't think the name matters much since it's static, but xslt_log() would be fine.